pkgs/development/tools/misc/unixbench/common.patch

```diff
@@ -0,0 +1,71 @@
+diff --git a/UnixBench/Run b/UnixBench/Run
+index 34d2c72..60a355a 100755
+--- a/UnixBench/Run
++++ b/UnixBench/Run
+@@ -81,23 +81,23 @@ my $longIterCount = 10;
+ my $shortIterCount = 3;
+ 
+ # C compiler to use in compilation tests.
+-my $cCompiler = 'gcc';
++my $cCompiler = 'cc -w';
```

Comment:
```suggestion
+my $cCompiler = '${targetPackages.stdenv.cc.targetPrefix}cc -w';
```

🤯

Reply:
Don't worry, they are already here:
https://github.com/NixOS/nixpkgs/blob/a8dc1d1888d8eeca36a410cce16e58b30c8695c8/pkgs/development/tools/misc/unixbench/default.nix#L91

Reply:
Yes, but `cc` may not refer to anything when you do `host != target`.

Reply:
> Yes, but `cc` may not refer to anything when you do `host != target`.

But now it should be host = target

Reply:
> > Yes, but `cc` may not refer to anything when you do `host != target`.
> 
> But now it should be host = target

`host != build`, sorry.

Reply:
> `target != build`, sorry.

You mean, if I cross compile a cc package, which serves as the native compiler on that platform, it still doesn't contain a `cc`? I've never did that anyway.

Reply:
I think my argument is wrong, after checking `pkgsCross.aarch64-multiplatform.targetPackages.stdenv.cc`. Might still be valid on some exotic platforms, though.

Reply:
> I think my argument is wrong, after checking `pkgsCross.aarch64-multiplatform.targetPackages.stdenv.cc`. Might still be valid on some exotic platforms, though.

Then we can still correct it when someone posted an issue I think. Dealing with a lot of foreign targets I've never touched seems exhausting.